src/terraform/devcontainer-feature.json

```diff
@@ -32,6 +32,22 @@
             ],
             "default": "latest",
             "description": "Terragrunt version"
+        },
+        "tfsec": {
+            "type": "string",
+            "proposals": [
+                "latest"
+            ],
+            "default": "latest",
+            "description": "TFSec version"
+        },
+        "terraform-docs": {
+            "type": "string",
+            "proposals": [
+                "latest"
+            ],
+            "default": "latest",
```

Comment:
If you decide to covert these two options to `boolean`, then I wonder if the defaults should be `false` 🤔 

Deferring this decision to @craiglpeters